def fun():
    print("In fun")
def fun(x):
    print("In run-x")

fun(5)
fun(10)
#fun() - give error because it will consider near fun method , and nears fun method has 1 parameter

